addon/plugins/snippet-plugin/utils/rdfa-predicate.ts

```diff
@@ -0,0 +1,50 @@
+import { PNode } from '@lblod/ember-rdfa-editor';
+import { AttributeProperty } from '@lblod/ember-rdfa-editor/addon/core/rdfa-processor';
+import { getProperties } from '@lblod/ember-rdfa-editor/utils/_private/rdfa-utils';
+import { getSnippetIdFromUri } from '@lblod/ember-rdfa-editor-lblod-plugins/plugins/snippet-plugin';
+
+export const SNIPPET_LIST_RDFA_PREDICATE = 'say:snippetListIds';
```

Comment:
As the triples are singular now, maybe it is better to use the `say:allowedSnippetList` as @abeforgit suggested?